includes/class-sensei.php

```diff
@@ -388,7 +388,7 @@ public function initialize_global_objects() {
 		// data will be sent.
 		$this->usage_tracking->schedule_tracking_task();
 
-		Sensei_Blocks::instance()->init();
+		Sensei_Blocks::instance()->init( $this );
```

Comment:
I needed to send the instance to the method, otherwise, we'd go through an infinite loop calling the `init` function. We can always handle it like that, or in the future we could rethink this architecture. I didn't want to change it now because this code is temporary and a refactor in this part could be very sensitive.